style.css

```diff
@@ -1 +1,69 @@
-/*  Add your styling here */
+
+/* Universal styles */
+
+* {
+    font-family: 'Pacifico', cursive;
+    background-color: rgb(188, 243, 226);
+}
+
+ul {
+    display: flex;
+    justify-content: center;
+}
```

Comment:
Be careful when styling a generic HTML tag with a specific rule such as `justify-content: center;`
You may well have instance of lists in your site which you don't want to style in the same way. To avoid this, add a class to the element and style that instead.